src/components/partnership/partnership.service.ts

```diff
@@ -447,6 +529,29 @@ export class PartnershipService {
       throw new NotFoundException('Could not find partnership');
     }
 
+    let mouStart = null;
+    let mouEnd = null;
+
+    // if user has access to project mou and there is no partnership override
+    if (result.projectMouStart && result.mouStartOveride !== null) {
```

Comment:
It's better to use double equals instead of tripe for comparing to null because that matches null & undefined which are used pretty interchangeably. 

It's a little weird that the checks aren't consistent here either. The first is _truthy_ and the second is not nullable.
Either is probably fine...
```suggestion
    if (result.projectMouStart && result.mouStartOveride) {
```
```suggestion
    if (result.projectMouStart != null && result.mouStartOveride != null) {
```

Reply:
I forgot I needed to refactor this now that I'm returning the project permissions. The conditionals are now calculated purely with permissions.